packages/scene-composer/src/hooks/useMatterportTags.ts

```diff
@@ -200,60 +124,47 @@ const updateTag = async (
         position: [item.anchorPosition.x, item.anchorPosition.y, item.anchorPosition.z],
       },
       components: components,
-      properties: {
-        [SceneNodeRuntimeProperty.LayerIds]: layerId && dynamicSceneEnabled ? [layerId] : undefined,
-      },
     },
     false,
-    !isUpdate,
   );
 };
 
 type AddTagInputs = {
-  layerId?: string;
-  sceneRootId?: string;
   id: string;
   item: MattertagItem | TagItem;
 };
 type UpdateTagInputs = {
-  layerId?: string;
-  sceneRootId?: string;
   ref: string;
   node: ISceneNodeInternal;
   item: MattertagItem | TagItem;
 };
 
 const useMatterportTags = (): {
-  handleAddMatterportTag: (inputs: AddTagInputs) => Promise<void>;
-  handleUpdateMatterportTag: (inputs: UpdateTagInputs) => Promise<void>;
-  handleRemoveMatterportTag: (nodeRef: string) => Promise<void>;
+  handleAddMatterportTag: (inputs: AddTagInputs) => void;
+  handleUpdateMatterportTag: (inputs: UpdateTagInputs) => void;
+  handleRemoveMatterportTag: (nodeRef: string) => void;
 } => {
   const sceneComposerId = useSceneComposerId();
   const appendSceneNode = useStore(sceneComposerId)((state) => state.appendSceneNode);
   const updateSceneNodeInternal = useStore(sceneComposerId)((state) => state.updateSceneNodeInternal);
   const removeSceneNode = useStore(sceneComposerId)((state) => state.removeSceneNode);
-  const dynamicSceneEnabled = useDynamicScene();
 
   const handleAddMatterportTag = useCallback(
     async (inputs: AddTagInputs) => {
-      await addTag(dynamicSceneEnabled, appendSceneNode, inputs);
+      addTag(appendSceneNode, inputs);
     },
     [appendSceneNode],
   );
 
   const handleUpdateMatterportTag = useCallback(
     async (inputs: UpdateTagInputs) => {
-      await updateTag(dynamicSceneEnabled, updateSceneNodeInternal, inputs);
+      updateTag(updateSceneNodeInternal, inputs);
```

Comment:
Confirmed reasoning on removing promises with @sheilaXu. This no longer requires a call to API, which eliminates the need for using a promise.